README.md

```diff
@@ -115,12 +115,12 @@ This will return a token:
 }
 ```
 
-This access token has the required scope of `data_receiver/notify` to access the endpoint :
+This access token has the required scope of `data/publish` to access the endpoint :
 ```json
 {
   "sub": "len",
   "nbf": 1669826698,
-  "scope": "data_receiver/notify",
+  "scope": "data/publish",
```

Comment:
This is different to the ADR which has `events/publish` is that deliberate?

Reply:
nope, typo